internal/json/gojson.go

```diff
@@ -0,0 +1,14 @@
+//go:build !(sonic && amd64 && avx512) && !stdjson
```

Comment:
Could you please explain why it is like this `sonic && amd64 && avx512`? I am not familiar with sonic, but in README there is nothing about `avx512`

Reply:
This is my mistake. When I previously used Sonic, it required AVX, and I'm not sure if it no longer needs AVX now. 

I have replaced the flag with only sonic because Sonic internally includes judgments for the operating system, architecture, and instruction set. It will automatically fallback to encode/json when not supported. I believe we do not need to make this judgment.